### ISSUE TITLE
ListItem children for Thmutil Combobox 

### ISSUE THREAD

**Issue by dbengle** [wixtoolset/issues#5354]
There is currently no way to populate a combo box in a WiX Burn standard bootstrapper application.  Many times you would like to add an option to the Options page, but there are far too many choices to make radio buttons practical.

The ComboBox in the Wix schema does have a child element, ListItem, for populating the combo box.

As far as I can tell, there is no way to reasonably accomplish this currently.  What I do right now is have an edit box, where the user can enter a value by typing it in.  However this is far from ideal, because the user can enter something that isn't one of the valid choices, and it isn't very user friendly.

The ThmUtil Combobox could have a ListItem child similar to the ComboBox in the WiX schema.  Like the Editbox, the Combobox name could be bound to a variable.   The variable could be used to retrieve and set the combo box selection, according to the Value in the ListItem.

In other words, if I set the variable to "123", it would select the combo box item with a value of "123".  If a user changed the combo box selection to an item with a value of "456", then the variable would be changed to "456".

**Comment by sarayukrishna** [comment]
Im able to add a combo box using burn bootstrapper with wix toolset 11. But not able to populate the values. Please do needful.
